lib/phoenix_live_view/igniter/upgrade_to_1_1.ex

```diff
@@ -214,20 +208,9 @@ defmodule Phoenix.LiveView.Igniter.UpgradeTo1_1 do
           ["NODE_PATH"],
           ~s<"[Path.expand("../deps", __DIR__), Mix.Project.build_path()])>,
           fn zipper ->
-            original_zipper = zipper
-
-            {:ok, zipper} =
-              Igniter.Code.Common.replace_code(zipper, "[Mix.Project.build_path()]")
-              |> Igniter.Code.List.prepend_to_list(
-                Code.string_to_quoted!(Igniter.Util.Debug.code_at_node(zipper))
-              )
-
-            # when we just return the ok tuple from above, things break in a very weird way
-            {:ok,
-             Igniter.Code.Common.replace_code(
-               original_zipper,
-               Igniter.Util.Debug.code_at_node(zipper)
-             )}
+            zipper
```

Comment:
not sure what was breaking from before, but this didn't mess up the tests for me?